.helm/build-images.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -e
+
+SCRIPT_DIR=$(realpath "$(dirname "$0")")
+REPO_DIR=$(realpath "$SCRIPT_DIR"/..)
+
+# docker hub
+registry_uri=https://index.docker.io/v1/
+not_logged_in="echo \"You need to login to the docker registry $registry_uri\" && exit 1"
+jq -e ".auths | .\"${registry_name}\" | .auth" < ~/.docker/config.json || $not_logged_in
+
+source "$SCRIPT_DIR"/.env
+
+if [ -z "$docker_hub_account" ] \
+    || [ -z "$version" ] \
+    ; then
+    echo "Please specify the needed env variables in $SCRIPT_DIR/.env"
+    echo "An example can be seen here:"
+    cat $SCRIPT_DIR/.env-example
+    exit 1
+fi
+
+frontend_image_tag="${docker_hub_account}/ecamp3-frontend:${version}"
+docker build "$REPO_DIR" -f "$REPO_DIR"/.docker-hub/frontend/Dockerfile -t "$frontend_image_tag"
+docker push "$frontend_image_tag"
+
+api_image_tag="${docker_hub_account}/ecamp3-php:${version}"
```

Comment:
I agree to have any renaming in a separate PR. Still would have preferred to have it consistent with the rest of the build & deployment chain.

As to the proposed rename: I see the point, that `ecamp3-api` is not ideal, but neither is `ecamp3-php`. For all the other images and services, we use a description of what each service is doing (frontend, api, print, etc.) and not about the technology used (php, nuxt, node, etc.), except for the ones that expose a bare/unaltered service (browserless, postgres, etc.). Otherwise we run into name conflict, as soon as we decide to implement a second service in PHP. What about `ecamp3-api-php` and `ecamp3-api-caddy`?

Reply:
ecamp3-api-php and ecamp3-api-caddy sound good to me.

Do you still prefer the consistent naming?
Then i would configure the api image outside of the loop

Reply:
You can decide. Until there's a second approver, there's still some time to clean it up 😄 
Otherwise you can also merge